Resources/doc/reference/form_types.rst

```diff
@@ -370,6 +370,9 @@ According the choice made only associated fields are displayed. The others field
 map
   Associative array. Describes the fields that are displayed for each choice.
 
+**TIP**: To mask a subfield replace `.` by `__`.
```

Comment:
Oh indeed it's sonata. But when we choose to change dots into double underscore, the assumption is that no one is going to use double underscores in their fields, ever, isn't it?